### DISCUSSION TITLE
Improve confirmation question for `r`

### DISCUSSION THREAD

**Discussion by tkossak** [leo-arch/clifm#246] [Ideas]
See this action in clifm:

```
<0> $ r 2
rm: remove 1 argument recursively?
```

What do you think about modifying the question, so it actually includes the names of the files, so we are sure we didn't written eln by mistake?

End result would look like:
```
<0> $ r 2
rm: arguments to remove recursively: file1.txt
Are you sure?
```

It would list all the files (maybe truncated if the list was too long), before asking if we are sure to remove them.
If there are confirmations for other commands, it could be useful to modify them too.

**Comment by leo-arch** [comment]
It can be done. Currently, that question is triggered by **rm**(1) (via the `-I` parameter) and not by **clifm** itself. I guess I can remove this parameter and replace the question whenever it's appropriate. I'll see what I can do.

**Comment by leo-arch** [comment]
Done. `r` will prompt now in the following cases (provided `-f`/`--force` is not used):

1. At least one directory
2. Three or more files
3. At least one file is a non-explicitly-expanded ELN.

In all the 3 cases, the list of files to be removed is printed before asking for confirmation.

Please give it a try.

**Comment by tkossak** [comment]
Works, thank you very much. Now I have peace of mind when removing files by eln :)
